Makefile

```diff
@@ -56,12 +56,16 @@ test-short: ## Execute the Golang's tests for updatecli
 	go test ./... -short
 
 .PHONY: test-e2e
-test-e2e: ## Execute updatecli end to end tests
+test-e2e: $(VENOM_VAR_binpath)/updatecli ## Execute updatecli end to end tests
 	@echo "==\nUsing the following updatecli binary (from variable 'VENOM_VAR_binpath'): $(VENOM_VAR_binpath)/updatecli\n=="
 	@test -n "$$GITHUB_TOKEN" || { echo "Undefined required variable 'GITHUB_TOKEN'"; exit 1; }
 	@test -n "$$GITHUB_ACTOR" || { echo "Undefined required variable 'GITHUB_ACTOR'"; exit 1; }
 	time venom run e2e/venom.d/* --output-dir ./e2e --format yaml
 
+# Not PHONY
+$(VENOM_VAR_binpath)/updatecli:
+	go build -o $(VENOM_VAR_binpath)/updatecli
```

Comment:
Thanks for documenting. 
I must admit I always run `go build -o bin/updatecli && ./bin/updatecli diff --config e2e/updatecli.d`

Otherwise running `make build test-e2e` is super slow because it cross compile